tests/conftest.py

```diff
@@ -1,5 +1,5 @@
 import asyncio
-from typing import Any, AsyncGenerator, Callable, Generic, Optional, Type, Union
+from typing import Any, AsyncGenerator, Callable, Generic, Optional, Tuple, Type, Union
```

Comment:
The Tuple import is unused and makes the CI complain 😬 Remove this and we should be good to go!

Reply:
Done!